applications/crossbar/doc/internationalization/numbers.md

```diff
@@ -83,33 +83,33 @@ This is a set of regexes to group numbers by type and are not used for routing.
 
     "classifiers":{
         "tollfree_us":{
-            "regex":"^\\+1(800|888|877|866|855)\\d{7}$",
+            "regex":"^\\+1((800|888|877|866|855)\\d{7})$",
```

Comment:
I think the inner capture groups should be non-capturing: `(?:800|888|877|866|855)` in this case., no?